### PR TITLE
feat: Customize hd_path [APE-624]

### PR DESCRIPTION
**Pull request by bilbeyt**
Add option hd_path for config

fixes: #

### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

File: ape_ganache/provider.py [hunk 1]
```diff
@@ -138,6 +138,10 @@ def mnemonic(self) -> str:
     def number_of_accounts(self) -> int:
         return self._test_config.number_of_accounts
 
+    @property
+    def hd_path(self) -> str:
+        return self._test_config.hd_path
+
```
Comment (NotPeopling2day) [inline ape_ganache/provider.py:144]: If it's optional, maybe something like this?

```suggestion
    @property
    def hd_path(self) -> str:
        return getattr(self._test_config, "hd_path", "m/44'/60'/0'")

```

Comment (bilbeyt) [reply ape_ganache/provider.py:144]: It has a default value already in test_config as i updated also ape version. It will just work.

Comment (NotPeopling2day) [reply ape_ganache/provider.py]: Apologies, I now see the merged PR in core.

**Review by NotPeopling2day — APPROVED** [review]
LGTM

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**